tests/cmds/test_legal_hold.py

```diff
@@ -575,6 +629,28 @@ def test_list_with_csv_format_returns_no_response_when_response_is_empty(
     assert "Matter ID,Name,Description,Creator,Creation Date" not in result.output
 
 
+def test_events_shows_events_that_respect_type_filters(
+    runner, cli_state, get_all_events_success, all_events_response
+):
+
+    cli_state.sdk.legalhold.get_all_events.return_value = all_events_response
+    result = runner.invoke(
+        cli, ["legal-hold", "events", "--event-type", "HoldCreated"], obj=cli_state
+    )
+
+    assert "564564654566" in result.output
```

Comment:
From the test alone, it is not immediately clear why these values are expected.
Maybe a comment saying `// From `all_events_response` would help

Reply:
Defined variables at the top here. I don't love the names

```
OLDER_LEGAL_HOLD_CREATED_EVENT = "564564654566"
NEWER_LEGAL_HOLD_MEMBERSHIP_EVENT = "74533457745"
```

Reply:
The names are fine..

but alternatively they could be called 
`_CREATE_EVENT_ID` and `_MEMBERSHIP_EVENT_ID`

Reasons:
1.) they are already in a legal hold module so don't need the prefix,
2.) they can be internal since nothing should be importing or using them outside of this test module, and 
3.) ending with ID makes it more apparent that it's not a JSON object

Reply:
I took your suggestions. Thanks!